src/start-client.jsx

```diff
@@ -12,8 +12,22 @@ import '~/theme';
 import configureStore from '@plone/volto/store';
 import { Api, persistAuthToken, ScrollToTop } from '@plone/volto/helpers';
 
+import * as Sentry from '@sentry/browser';
+
 export const history = createBrowserHistory();
 
+if (__SENTRY__) {
+  Sentry.init({ dsn: __SENTRY__.SENTRY_DSN });
```

Comment:
It would be awesome to also add more sentry configuration support here: I would add release, environment, serverName, maxBreadcrumbs support, which look easy to implement and useful to have, in some situations.
https://docs.sentry.io/error-reporting/configuration/?platform=javascript

Reply:
We already have the possibility to set the tags and addititonal data, using the env variables SENTRY_FRONTEND_CONFIG and SENTRY_BACKEND_CONFIG.
The documentation will be updated soon.